test/experimental/test_transforms.py

```diff
@@ -13,6 +18,50 @@
 
 
 class TestTransforms(TorchtextTestCase):
+    def test_sentencepiece_processor(self):
+        model_path = get_asset_path('spm_example.model')
+        spm_transform = sentencepiece_processor(model_path)
+        jit_spm_transform = torch.jit.script(spm_transform.to_ivalue())
+        test_sample = 'SentencePiece is an unsupervised text tokenizer and detokenizer'
+        ref_results = [15340, 4286, 981, 1207, 1681, 17, 84, 684, 8896, 5366,
+                       144, 3689, 9, 5602, 12114, 6, 560, 649, 5602, 12114]
+        self.assertEqual(spm_transform(test_sample), ref_results)
+        self.assertEqual(jit_spm_transform(test_sample), ref_results)
+        self.assertEqual(spm_transform.decode(ref_results), test_sample)
+        self.assertEqual(jit_spm_transform.decode(ref_results), test_sample)
+
+    def test_sentencepiece_tokenizer(self):
+        model_path = get_asset_path('spm_example.model')
+        spm_tokenizer = sentencepiece_tokenizer(model_path)
+        jit_spm_tokenizer = torch.jit.script(spm_tokenizer.to_ivalue())
+        test_sample = 'SentencePiece is an unsupervised text tokenizer and detokenizer'
+        ref_results = ['\u2581Sent', 'ence', 'P', 'ie', 'ce', '\u2581is',
+                       '\u2581an', '\u2581un', 'super', 'vis', 'ed', '\u2581text',
```

Comment:
Why do we use "\u2581an" when in other tests we use the actual UTF-8 character?

Reply:
Offline discussion: This is the whitespace symbol. How does the user know that this is the case?

Reply:
Updated the doc of SentencepieceTokenizer transform.